airflow/providers/databricks/operators/databricks.py

```diff
@@ -246,6 +246,7 @@ class DatabricksSubmitRunOperator(BaseOperator):
 
     # Used in airflow.models.BaseOperator
     template_fields: Sequence[str] = ('json',)
+    template_ext: Sequence[str] = ('json',)
```

Comment:
You should add it as ".json" not "json"

Reply:
@potiuk , I'll update PR with a '.json' extension.